### DISCUSSION TITLE
Metadata question (Sharing Label)

### DISCUSSION THREAD

**Discussion by olderthantheinternet** [meisnate12/Plex-Meta-Manager#21] [Q&A] [answered]
Looking at https://github.com/meisnate12/Plex-Meta-Manager/wiki/Metadata-Attributes

It mentions LABELS, does that include the **Sharing Label**
And when creating Collections, can a **Sharing Label** be set for each item in the Collection?

**Comment by meisnate12** [comment, marked as answer]
yes that sets the sharing label but not there's no way to mass edit it via a collection i could add that just submit a feature request

**Reply by olderthantheinternet** [reply]
So currently it can sets the Share label for just the collection, but not the items inside the collection.
Is that what you are saying ?

**Reply by meisnate12** [reply]
you can use the metadata attribute to set labels per item in your library but it has nothing to do with what collection they're in. Also as of right now you cannot set the sharing label of a collection just of an individual movie or show.
